typescript/introduction/tsc-interfaces-classes/tsc-classes-intro.md

```diff
@@ -0,0 +1,77 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Classes Intro
+
+---
+## Content
+
+Classes represent the building blocks of the style of programming called *object-oriented (class-based) programming*, or OOP[1]. 
```

Comment:
This statement doesn't seem right. object-oriented is not exactly *class based* but rather making classes is a common way of implementing OO paradigm. JS uses functions (before ES6) to do OO

Reply:
So what this is supposed to say is that this is [class-based OOP](https://en.wikipedia.org/wiki/Class-based_programming).

JS uses [prototype-based](https://en.wikipedia.org/wiki/Prototype-based_programming) OOP.

Reply:
You're right that this statement isn't clear. How about just this @sourabhv?

```suggestion
Classes represent the building blocks of the style of programming called *object-oriented programming*, or OOP[1]. 
```